packages/tasks/src/local-apps.ts

```diff
@@ -326,6 +326,14 @@ export const LOCAL_APPS = {
 		displayOnModelPage: isLlamaCppGgufModel,
 		snippet: snippetLocalAI,
 	},
+	localchat: {
+		prettyLabel: "Local Chat",
+		docsUrl: "https://localchat.co",
+		mainTask: "text-generation",
+		displayOnModelPage: (model) => isMlxModel(model),
```

Comment:
```suggestion
		displayOnModelPage: isMlxModel,
```

I think that should suffice :)